examples/rps-9-web/index.js

```diff
@@ -93,4 +95,19 @@ class Attacher extends Player {
   render() { return renderView(this, AttacherViews); }
 }
 
-renderDOM(<App />);
+window.reach = reach;
```

Comment:
What's going on here?

Reply:
oops this isn't supposed to get commited, this is from running `patch` for the browser tests locally